.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-sudo: true
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
+  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travis-coveralls/master/travis-coveralls.sh
+script:
+    - bash -ex .travis-opam.sh
+    - if [ ! -z ${XS_COV} ]; then bash -ex travis-coveralls.sh || true; fi
```

Comment:
In the first version I was using `${XS_COV+1}` for a similar reason but it seemed too much. I agree with you, fix added